release-calendar/src/client/stylesheets/channelTable.scss

```diff
@@ -0,0 +1,91 @@
+
+button:focus {
+    outline: none !important;
+}
+
+body {
```

Comment:
Move `body` styling to the top level stylesheet.

Reply:
I moved it to `index.html`. Is that where you meant?

Reply:
It could work there but I meant `app.scss`. Having styles in both html and scss files would be difficult to maintain.